specification/compute/resource-manager/Microsoft.Compute/2017-10-15-preview/swagger/computeArtifactPublishingService.json

```diff
@@ -0,0 +1,870 @@
+          "SharedVMImages"
+        ],
+        "operationId": "SharedVMImages_CreateOrUpdate",
+        "description": "Creates or updates a shared VM image.",
+        "parameters": [
+		  {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }, 
+		  {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          }, 
+		  {
+            "$ref": "#/parameters/SharedVMImageNameParameter"
+          }, 
+		  {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }, 
+		  {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SharedVMImage"
+            },
+            "description": "SharedVMImage object supplied in the body of the Put SharedVMImage operation."
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/SharedVMImage"
+            }
+          },
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/SharedVMImage"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "get": {
+        "tags": [
+          "SharedVMImages"
+        ],
+        "operationId": "SharedVMImages_Get",
+        "description": "Gets information about a shared VM image.",
+        "parameters": [{
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }, {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          }, {
+            "$ref": "#/parameters/SharedVMImageNameParameter"
+          }, {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SharedVMImage"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "SharedVMImages"
+        ],
+        "operationId": "SharedVMImages_Delete",
+        "description": "Deletes a shared VM image.",
+        "parameters": [{
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }, {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          }, {
+            "$ref": "#/parameters/SharedVMImageNameParameter"
+          }, {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "If the shared VM image is deleted, this is an expected error code."
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+	 "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/sharedvmimages/{sharedVMImageName}/versions/{version}": {
```

Comment:
Is versions a proxy resource? If it is tracked, then PATCH support is missing. I believe this should not be tracked, but wanted to confirm